compose/bin/cache-clean

```diff
@@ -11,13 +11,13 @@ fi
 
 if [ "$1" == "--watch" ]; then
   # Kill duplicate watch process
-  WATCH_PID=$(bin/clinotty ps -eaf | grep "$CACHE_CLEAN --quiet --watch" | grep -v grep | awk '{print $2}')
+  WATCH_PID=$(bin/clinotty ps -eaf | grep "$CACHE_CLEAN -w -d /var/www/html" | grep -v grep | awk '{print $2}')
```

Comment:
I'd like to retain full flag params so it's more apparent as to what is going on (`--watch` rather than `-w`, etc.).

Reply:
This code has been refactored into a docker-compose run call so it's no longer applicable